syntax = "proto3";

package client;

message ClAskPossibleMoves {
  int32 id = 1;
  int32 index = 2;
}